network-api/networkapi/mozfest/templates/partials/mozfest_footer.html

```diff
@@ -1,43 +1,23 @@
-{% with wrapper_class="py-5" %}
+{% extends "partials/footer.html" %}
```

Comment:
the change to this file is mostly to make sure that all that duplicated code stays in the template I originally copied it all from. No need to have all of that here when really all we want is the left column for the newsletter signup.